Packs/GitLab/Integrations/GitLabv2/GitLabv2_description.md

```diff
@@ -2,7 +2,7 @@ To access the requested repository, you need to provide
  the username, repository name, API key, and project ID as parameters for the
  integration.
  
- To locate your personal GitLab API key (scopes can be chosen according to the integration readme ):
+ To locate your personal GitLab API key (see the needed scopes in the integration documentation - add the link to the right part ):
```

Comment:
Remove the second part for now. (the link part)

Reply:
removed